index.html

```diff
@@ -388,6 +380,21 @@ <h2 class="title">Contact me</h2>
             </div>
         </footer>
     </main>
+    <!-- Menu Modal -->
+    <div id="menu-modal" class="menu-modal">
+        <nav>
+            <ul id="nav-list" >
+                <li><a href="#headline">Portoflio</a></li>
+                <li><a href="#about">About</a></li>
```

Comment:
- [ ] Kindly add the about id to the right section in your page, currently when a user clicks it shows the works section 👍🏿

Reply:
Thanks 👍🏿